examples/ws_subscription.rs

```diff
@@ -25,28 +25,28 @@
 // DEALINGS IN THE SOFTWARE.
 
 use jsonrpsee::{
-	types::{traits::SubscriptionClient, v2::ParamsSer, Error, Subscription},
+	rpc_params,
+	types::{traits::SubscriptionClient, Error, Subscription},
 	ws_client::WsClientBuilder,
 	ws_server::{RpcModule, WsServerBuilder},
 };
 use std::net::SocketAddr;
 
-const NUM_SUBSCRIPTION_RESPONSES: usize = 10;
-
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+	const NUM_SUBSCRIPTION_RESPONSES: usize = 5;
```

Comment:
why move const into the scope of a function, instead of just using a let/5 in the loop

Reply:
You're right, this is dumb. Will fix.